lib/services.js

```diff
@@ -202,25 +204,25 @@ var mSvcConfigs = {
 		 * for the other services, because the buckets-postgres service
 		 * uses the same image as the postgres service.
 		 */
-		'imageName': 'manta-postgres'
+		'imageNames': ['mantav1-postgres', 'manta-postgres']
 	},
 	'boray': {
 		'oneach': true,
 		'probes': true,
 		'sharded': true,
-		'imageName': 'manta-boray'
+		'imageNames': ['mantav1-boray', 'manta-boray']
 	},
 	'electric-boray': {
 		'oneach': true,
 		'probes': true,
 		'sharded': false,
-		'imageName': 'manta-electric-boray'
+		'imageNames': ['mantav1-electric-boray', 'manta-electric-boray']
 	},
 	'marlin': {
 		'oneach': false,
 		'probes': false,
 		'sharded': false,
-		'imageName': 'manta-marlin'
+		'imageNames': ['manta-marlin']
```

Comment:
Might be worth a comment here explaining why this guy is special.

Reply:
Good point, doing that.